ftva_lab_data/tests.py

```diff
@@ -510,3 +513,53 @@ def test_search_finds_carrier_location_in_property_field(self):
             search_fields=["carrier_a_with_location"],
         )
         self.assertEqual(items.all()[0], self.item_basic)
+
+
+class ItemStatusTestCase(TestCase):
+    fixtures = ["sample_data.json", "item_statuses.json"]
```

Comment:
There's currently one test, and it does not use any of the sample records in the `sample_data.json` fixture.  As-is, that fixture doesn't need loading for this test case.

`match_record()` could be tested, but I think there's not much value to doing that.  You could create a `pd.Series` from a list of strings and test that way - no need to create a full dataframe or Excel file - but that's just confirming that database filter queries work, and I think we can trust that.

So, unless you want to add test(s) for `match_records()`, you don't need the `sample_data` fixture.